romanisim/wcs.py

```diff
@@ -188,9 +189,11 @@ def _radec(self, x, y, color=None):
             return r[0], d[0]
         else:
             if (np.ndim(x) != np.ndim(y)):
-                raise ValueError(f"np.ndim(x) != np.ndim(y) => {np.ndim(x)} != {np.ndim(y)}")
+                raise ValueError(
+                    f"np.ndim(x) != np.ndim(y) => {np.ndim(x)} != {np.ndim(y)}")
             elif (x.shape != y.shape):
-                raise ValueError(f"x.shape != y.shape => {x.shape} != {y.shape}")
+                raise ValueError(
```

Comment:
Do you want to add tests for these uncovered failures (to ensure things work right down the line)?